main.go

```diff
@@ -296,6 +297,8 @@ func defineFlagVar() *FlagVar {
 		"Indicates the failure max delay in seconds")
 	flag.DurationVar(&flagVar.cacheSyncTimeout, "cache-sync-timeout", defaultCacheSyncTimeout,
 		"Indicates the cache sync timeout in seconds")
+	flag.BoolVar(&flagVar.enableDomainNameVerification, "enable-domain-name-pinning", false,
```

Comment:
You are right - Since the watcher is by default off, we can set this to true! But I would definitely keep the feature-flag, to turn it off in simple test use-cases.